### ISSUE TITLE
WSDL files that don't contain 'wsdl' in the file are not imported correctly

### ISSUE THREAD

**Issue by ffaen** [getinsomnia/insomnia#4512]
### Expected Behavior

Importing a WSDL file (for example, `http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL`) will create a collection of endpoints to query

### Actual Behavior

`This file does not contain a valid specification`

```
Error: No importers found for file
    at convert (C:\Users\{snip}\AppData\Local\insomnia\app-2021.7.2\resources\app.asar\node_modules\insomnia-importers\dist\src\convert.js:30:11)
    at async importRaw (file:///C:/Users/{snip}/AppData/Local/insomnia/app-2021.7.2/resources/app.asar/bundle.js:90952:15)
    at async importUri (file:///C:/Users/{snip}/AppData/Local/insomnia/app-2021.7.2/resources/app.asar/bundle.js:90905:18)
    at async file:///C:/Users/{snip}/AppData/Local/insomnia/app-2021.7.2/resources/app.asar/bundle.js:67114:22
```

### Reproduction Steps

1. In the main menu, click 'Create'
2. 'Import From URL'
3. Paste in http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL
4. Error occurs

You can additionally generate your own WSDL using something like [Laminas SOAP Server Library](https://docs.laminas.dev/laminas-soap/auto-discovery/), which has the same issue

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

This issue has been mentioned previously: https://github.com/Kong/insomnia/issues/1103#issuecomment-749418281 but I haven't seen any open issue with this report. 

### Insomnia Version

2021.7.2

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Pro Version	21H1

### Installation method

download from insomnia.rest

### Last Known Working Insomnia version

n/a

**Comment by ffaen** [comment]
The culprit is most likely [here](https://github.com/Kong/insomnia/blob/develop/packages/insomnia-importers/src/importers/wsdl.ts#L94), as these documents don't have the string `wsdl:definition`. I'm unsure if simply changing this to `definition` will work as your importer library may not support that style of WSDL.

**Comment by WilhanAzevedo** [comment]
Realmente , @ffaen esta certo , no meu arquivo alterei `definition` para `wsdl:definition` e com isso a importação funcionou

**Comment by filfreire** [comment]
@ffaen thank you for reporting this!

This issue where you might get an error like:

```
Error: No importers found for file
```

should be solved in our latest beta, [2022.3.0-beta.3](https://github.com/Kong/insomnia/releases/tag/core%402022.3.0-beta.3).

@ffaen  @WilhanAzevedo, regarding using `definition` vs. `wsdl:definition` - according to the [WSDL specification](https://www.w3.org/TR/wsdl.html#_document-s), it should be `wsdl:definition` or `wsdl:definitions`.

Feel free to re-open the issue if you still have problems.

**Comment by ffaen** [comment]
Hi @filfreire, thanks for taking a look at this.

I'm not sure the specification requires you to use `wsdl`, as in your link you can see an example WSDL without the prefix in [section 3.1](https://www.w3.org/TR/wsdl.html#_soap-e) examples 3,4 and 5.

I'm pretty sure the prefix just namespaces the key, you can use whatever as long as you have the correct `xmlns` bit at the top (not providing a key means it's in the 'root' namespace, like the example I provided in the original report)

Here are some more examples of `definition(s)` being used in the wild:

- https://www.tutorialspoint.com/wsdl/wsdl_example.htm
- https://www.w3.org/2001/04/wsws-proceedings/uche/wsdl.html
- https://cs.au.dk/~amoeller/WWW/webservices/wsdlexample.html

Hopefully this will help convince that it's suitable for insomnia to allow for this.

**Comment by RobertKlohr** [comment]
I have a wsdl that exhibits the exact same error as the one @ffaen linked to except mine does contain a wsdl:definitions tag.

@filfreire I have tested this file with the latest gm build [2022.1.1](https://github.com/Kong/insomnia/releases/tag/core%402022.1.0) and the [2022.3.0-beta.3](https://github.com/Kong/insomnia/releases/tag/core%402022.3.0-beta.3).

This makes me think the issue is either unrelated to the wsdl:definitions tag but, at least in my case, how the xmlns is being interpreted. I will attach the two files for comparison and testing.

The original file that does not import: [Sample-error-wsdl.xml.txt](https://github.com/Kong/insomnia/files/8497932/Sample-error-wsdl.xml.txt)

The original file with one manual edit to add the following to the beginning of the definitions tag that does import: [Sample-imports-wsdl.xml.txt](https://github.com/Kong/insomnia/files/8497942/Sample-imports-wsdl.xml.txt)
``
xmlns="http://schemas.xmlsoap.org/wsdl/"
``

The second file validates in all tools I have at my disposal. The first one validates in most tools but fails to import in Insomnia and also fails to validate using [https://www.wsdl-analyzer.com/](https://www.wsdl-analyzer.com/). Maybe there are interpreters that are more strict than others?

@filfreire Is this a case where the wsdl I am using is not actually valid but works in some tools it needs to be fixed, we re-open and rename this issue, or we create a new issue?

**Comment by x0rld** [comment]
I also can't import some WSDL file with the wsdl:definition and it's validate using wsdl-analyzer.com and I can open them with SOAPUI  with the version [2.22.7.2](https://github.com/Kong/insomnia/releases/tag/core%402022.7.2)

**Comment by Endogen** [comment]
Same thing here. Can't import a valid WSDL file. Interestingly enough i can use a URL to import that same content

**Comment by Endogen** [comment]
> @ffaen thank you for reporting this!
> 
> This issue where you might get an error like:
> 
> ```
> Error: No importers found for file
> ```
> 
> should be solved in our latest beta, [2022.3.0-beta.3](https://github.com/Kong/insomnia/releases/tag/core%402022.3.0-beta.3).
> 
> @ffaen @WilhanAzevedo, regarding using `definition` vs. `wsdl:definition` - according to the [WSDL specification](https://www.w3.org/TR/wsdl.html#_document-s), it should be `wsdl:definition` or `wsdl:definitions`.
> 
> Feel free to re-open the issue if you still have problems.

I still have that issue in version 2023.4.0
https://dpaste.org/SUJrQ
